packages/stable/src/__tests__/testUtils.ts

```diff
@@ -186,6 +183,99 @@ const getFileCreateArgs = (
 
   return { postfix, fileContent, sourceCreatedTime, localFileMeta };
 };
+export function setupMockableClientForIntegrationTests() {
+  const client = setupClient(process.env.COGNITE_BASE_URL);
+  return client;
+}
+export function setupLoggedInClientForUnitTest() {
+  return new CogniteClient({
+    appId: 'JS SDK integration tests (beta)',
+    baseUrl: process.env.COGNITE_BASE_URL,
+    project: process.env.COGNITE_PROJECT as string,
+    getToken: () =>
+      login().then((account) => {
+        return account.access_token;
+      }),
+  });
+}
+
+export function setupMockableClientForUnitTest() {
+  const client = setupLoggedInClientForUnitTest();
+  return client;
+}
```

Comment:
This function doesn't nothing? It's just a proxy to `setupLoggedInClientForUnitTest`